src/service_client/assisted_service_api.py

```diff
@@ -470,11 +482,12 @@ def get_hosts_by_role(self, cluster_id: str, role, hosts=None):
         return nodes_by_role
 
     def get_api_vip(self, cluster_info: dict, cluster_id: str = None):
-        cluster = cluster_info or self.cluster_get(cluster_id)
+        cluster = cluster_info or self.cluster_get(cluster_id).to_dict()
         api_vip = cluster.get("api_vip")
-        user_managed_networking = cluster.get("user_managed_networking")
+        platform_type = cluster.get("platform", {}).get("type")
+        is_platform_network_managed = platform_type == consts.Platforms.OCI or platform_type == consts.Platforms.NONE
 
-        if not api_vip and user_managed_networking:
+        if not api_vip and is_platform_network_managed:
```

Comment:
In my opinion, `is_platform_network_managed` is misleading. It like it means the platform manages the network which is the opposite, we expect the network to be setup upfront.

Reply:
@adriengentil any suggestions? We also have this in assisted-service feature-support API

Reply:
`externaly_managed_networking` but I'm biased :sweat_smile: 
`infrastructure_managed_networking`